src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -190,6 +190,7 @@ public void setAssessmentDueDate(ZonedDateTime assessmentDueDate) {
     /**
      * Checks if the assessment due date is in the past. Also returns true, if no assessment due date is set.
      */
+    @JsonIgnore
```

Comment:
Without the `@JsonIgnore` a property `assessmentDueDateOver` will be serialized and sent to the client. We could also use this property in the client instead of calculating the `isAssessmentDueDateOver` in the different components over and over again, but it would reflect the status at the time of serializing the JSON in the server. Maybe some time has passed since then and the time of the assessment due date has passed in the mean time.